.changeset/honest-guests-wash.md

```diff
@@ -0,0 +1,5 @@
+---
+"@medusajs/medusa": patch
+---
+
+Introduces a model and migration to create category table that can be nested
```

Comment:
suggestion(non blocking): We generally start with something like `feat(product-category): ...`